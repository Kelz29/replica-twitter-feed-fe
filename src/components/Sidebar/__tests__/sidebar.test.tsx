import SidebarWithHeader from "../index";

describe("Sidebar with header Test", () => {
  it("renders successfully", () => {
    expect(SidebarWithHeader).toMatchSnapshot();
  });
});
