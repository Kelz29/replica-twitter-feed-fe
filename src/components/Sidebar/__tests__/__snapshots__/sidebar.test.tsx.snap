// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar with header Test renders successfully 1`] = `[Function]`;
